desc/objectives/_geometry.py

```diff
@@ -96,16 +100,18 @@ def build(self, eq, use_jit=True, verbose=1):
         self._args = get_params(self._data_keys)
 
         timer = Timer()
-        if verbose > 0:
-            print("Precomputing transforms")
+        logging.info("Precomputing transforms")
         timer.start("Precomputing transforms")
 
         self._profiles = get_profiles(self._data_keys, eq=eq, grid=self.grid)
         self._transforms = get_transforms(self._data_keys, eq=eq, grid=self.grid)
 
         timer.stop("Precomputing transforms")
-        if verbose > 1:
-            timer.disp("Precomputing transforms")
+        timer.disp("Precomputing transforms")
+
+        if self._normalize:
```

Comment:
this if block does not belong here, this is the reason why the tests are failing. Aspect ratio is a unitless quantity, so it does not need normalization.